File: strings/notes.py
#Gavin Woodhouse, Strings Notes Python
# Words in programming (data type) "" ''

name = input("What is your name?\n").strip().capitalize()

print(f"Hello {name} welcome to my program.")

sentence = "The quick brown fox"
print(sentence.find("fox"))

word = sentence.find("fox")
#print(sentence[4:9])
#print(sentence[word:word+3])
#print(len(sentence))

percent = 89
print(f"Your grade is {percent:.1f}")

File: strings/notes.c
//Gavin Woodhouse, Template
#include <stdio.h>
#include <string.h>

char name[20];

int main(void){
    //printf("Please tell me your full name:\n");
    //scanf("%s", name);
    //fgets(name, 20, stdin);
    //printf("Hello %s, welcome to my program", name);
    //char sentence[] = "The quick brown fox jumps over the lazy dog";
    //printf("%c\n", sentence[16]);
    //printf("%d\n", strlen(sentence));
    char one[] = "Hello ";
    char two[] = "World!";
    char three[] = "This is my program. ";
    two[5] = '?';
    strcat(one, two);
    printf("%s\n", one);
    strcat(three, one);
    printf("%s\n", three);
    return 0;
}